internal/service/api/handlers/get_permissions.go

```diff
@@ -18,13 +18,13 @@ func GetPermissions(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var userIds []int64
-	if request.UserId != nil {
-		userIds = append(userIds, *request.UserId)
+	var usernames []string
+	if request.Username != nil {
+		usernames = append(usernames, *request.Username)
```

Comment:
You need to have both userIds and usernames filters, because FE passes userId when makes requests from verified user page